api/src/api/resource.js

```diff
@@ -162,7 +162,10 @@ router.get(
     if (!('totalData' in resources[0])) {
       const resourcesList = resources;
       resources = [];
-      resources[0] = { totalData: resourcesList };
+      resources[0] = {
+        totalData: resourcesList,
+        totalCount: [resourcesList.length],
```

Comment:
what's totalCount used here for?

Reply:
That's to make every response from this API return the same data, so that on the frontend it gets everything it expects. So technically the total count here isn't used for mapview, but we could use it in the future for pagination